crates/nu-command/src/input_handler.rs

```diff
@@ -7,6 +7,28 @@ pub trait CmdArgument {
     fn take_cell_paths(&mut self) -> Option<Vec<CellPath>>;
 }
 
+/// Arguments with only cell_path.
+///
+/// If commands is going to use `operate` function, and it only required optional cell_paths
+/// Using this to simplify code.
+pub struct ArgumentsCP {
```

Comment:
I think the name is a bit obscure: 
Maybe `CellPathArgs` or `CellPathOnlyArgs`?

Reply:
Thanks!  I like the name `CellPathOnlyArgs`